Library/Project/Author/AuthorList.aspx

```diff
@@ -0,0 +1,38 @@
+﻿<%@ Page Title="Author List" Language="C#" MasterPageFile="~/Site.Master" AutoEventWireup="true" CodeBehind="AuthorList.aspx.cs" Inherits="Library.AuthorList" %>
+
+<%@ import namespace="System.Data" %>
+
+<asp:Content ID="Content1" ContentPlaceHolderID="main" runat="server">
+
+    <h2>Authors</h2>
+
+    <div>
+        <asp:hyperlink runat="server" class="btn btn-primary mb-2" NavigateUrl="~/Project/Author/AuthorAdd.aspx">Add New Author</asp:hyperlink>
+    </div>
+
+    <asp:repeater id="Authors" runat="server" itemtype="DataRow">
+        <headertemplate>
+            <table class="table border table-sm">
+                <thead class="thead-dark">
+                    <tr>
+                        <th>First Name</th>
+                        <th>Last Name</th>
+                        <th>&nbsp;</th>
+                    </tr>
+                </thead>
+        </headertemplate>
+        <itemtemplate>
+            <tbody>
```

Comment:
You should be able to put your opening `<tbody>` element in the header template and the closing `</tbody>` element in the footer template.